spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
```

Comment:
- [ ] Hi @Wils-FOURR-speed, as requested by the previous reviewer, kindly make sure app is working well so that your tests can run we success. All your tests are still failing as you can see on the screenshot below:
![Screenshot from 2023-11-03 12-37-52](https://github.com/Wils-FOURR-speed/Blog-App-Project/assets/63087491/af008150-f51a-4cab-bf0d-043b40bc8d48)